scripts/update/jellyfin.sh

```diff
@@ -1,93 +1,104 @@
-        sed -r 's#<RequireHttps>false</RequireHttps>#<RequireHttps>true</RequireHttps>#g' -i /etc/jellyfin/system.xml
-        sed -r 's#<CachePath>(.*)</CachePath>#<CachePath>/var/cache/jellyfin</CachePath>#g' -i /etc/jellyfin/system.xml
-        sed -r 's#<MetadataPath>(.*)</MetadataPath>#<MetadataPath>/var/lib/jellyfin/metadata</MetadataPath>#g' -i /etc/jellyfin/system.xml
-        echo_warn "The Jellyfin ports have been moved to 8920 (SSL). Please update any applications that rely on the old random port"
-        #
-        if [[ -f /etc/nginx/apps/jellyfin.conf ]]; then
-            sed -r 's#proxy_pass https://127.0.0.1:(.*);#proxy_pass https://127.0.0.1:8920;#g' -i /etc/nginx/apps/jellyfin.conf
-            systemctl -q restart nginx.service
-        fi
-        #
-        rm_if_exists "/home/${username}/.config/Jellyfin"
-        rm_if_exists "/home/${username}/.cache/jellyfin"
-        rm_if_exists "/home/${username}/.aspnet"
-        echo_progress_done "Configs adjusted"
-    fi
-    #
-    if ! check_installed jellyfin; then
-        echo_info "Moving Jellyfin to apt-managed installation"
-        #
-        # Add the jellyfin official repository and key to our installation so we can use apt-get to install it jellyfin and jellyfin-ffmepg.
-        wget -q -O - "https://repo.jellyfin.org/$DIST_ID/jellyfin_team.gpg.key" | apt-key add - >>"${log}" 2>&1
-        echo "deb [arch=$(dpkg --print-architecture)] https://repo.jellyfin.org/$DIST_ID $DIST_CODENAME main" >/etc/apt/sources.list.d/jellyfin.list
-        #
-        # install jellyfin and jellyfin-ffmepg using apt functions.
-        apt_update #forces apt refresh
-        apt_install jellyfin jellyfin-ffmpeg
-        #
-        # Configure the new jellyfin service.
-        systemctl -q stop jellyfin.service
-        #
-        # Add the jellyfin user to the master user's group to use our ssl certs.
-        usermod -a -G "${username}" jellyfin
-        #
-        # Set the correct and required permissions of any directories we created or modified.
-        chown "${username}.${username}" -R "/home/${username}/.ssl"
-        chmod -R g+r "/home/${username}/.ssl"
-        #
-        # Set the default permissions after we have migrated our data
-        chown -R jellyfin:jellyfin "/etc/jellyfin"
-        chown jellyfin:root "/etc/jellyfin/logging.json"
-        chown jellyfin:adm "/etc/jellyfin"
-        chown -R jellyfin:jellyfin "/var/lib/jellyfin"
-        chown jellyfin:adm "/var/lib/jellyfin"
-        #
-        # Reload systemd and start the service.
-        systemctl -q daemon-reload
-        systemctl -q start jellyfin.service
-        #
-        echo_success "Jellyfin updated"
-    fi
+	# awaiting pull to remove
+	function dist_info() {
+		DIST_CODENAME="$(source /etc/os-release && echo "$VERSION_CODENAME")"
+		DIST_ID="$(source /etc/os-release && echo "$ID")"
+	}
```

Comment:
think you might want to remove this from this PR if it's already in the other one

Reply:
the function need to me merged first.